### ISSUE TITLE
Rename local variable names

### ISSUE THREAD

**Issue by Palollo** [mkaring/ConfuserEx#19]
Private atributes, properties and methods are renamed, also invocation to public methods are obfuscated with a proxy, but local variables are kept clear. Is there any reason? It seems easier. There is a configuration for that?

**Comment by mkaring** [comment]
The easiest way to handle local variables is stripping the names for those entirely. The names of local variables are just additional information and are not required in any way.

**Comment by mkaring** [comment]
I finally looked into that and the reason why the names of the local variables are not removed is pretty simple. The names are already removed by the C#/VB.NET compiler when compiling with the Release configuration.
Did you try to obfuscate a debug assembly?

**Comment by bprg** [comment]
I think _local variables_ name stored in pdb file.

**Comment by Palollo** [comment]
No, the assembly was compiled in Release mode and without PDB. 
It is .NET 2.0, maybe it is the reason? I will make a small test with .NET 4.6.
